plugins/strict-type-checker/README.md

```diff
@@ -0,0 +1,43 @@
+# blockly-plugin-strict-type-checker [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```

Comment:
Any reason not to match the naming guide with `@blockly/plugin-strict-type-checker`?

Also, should this be called a strict-connection-checker instead as it's overriding a connection checker.